src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/resources/WorkerResource.java

```diff
@@ -0,0 +1,115 @@
+package edu.kit.ipd.crowdcontrol.objectservice.rest.resources;
+
+import edu.kit.ipd.crowdcontrol.objectservice.crowdworking.PlatformManager;
+import edu.kit.ipd.crowdcontrol.objectservice.crowdworking.UnidentifiedWorkerException;
+import edu.kit.ipd.crowdcontrol.objectservice.database.operations.WorkerOperations;
+import edu.kit.ipd.crowdcontrol.objectservice.proto.Worker;
+import edu.kit.ipd.crowdcontrol.objectservice.proto.WorkerList;
+import edu.kit.ipd.crowdcontrol.objectservice.rest.Paginated;
+import edu.kit.ipd.crowdcontrol.objectservice.rest.exceptions.BadRequestException;
+import edu.kit.ipd.crowdcontrol.objectservice.rest.exceptions.NotFoundException;
+import spark.Request;
+import spark.Response;
+
+import static edu.kit.ipd.crowdcontrol.objectservice.rest.RequestUtil.*;
+
+/**
+ * Handles requests to worker resources.
+ *
+ * @author Niklas Keller
+ */
+public class WorkerResource {
+    private WorkerOperations operations;
+    private PlatformManager manager;
+
+    public WorkerResource(WorkerOperations operations, PlatformManager manager) {
+        this.operations = operations;
+        this.manager = manager;
+    }
+
+    /**
+     * @param request  request provided by Spark.
+     * @param response response provided by Spark.
+     *
+     * @return Worker if found.
+     */
+    public Worker identify(Request request, Response response) {
+        String identity;
+
+        try {
+            identity = manager.identifyWorker(request.params("platform"), request.queryMap().toMap());
```

Comment:
I think public static is fine, since the methods can useful in other pieces of code